docs/getting-started/security-ui.asciidoc

```diff
@@ -32,6 +32,29 @@ The navigation menu contains direct links and expandable groups, identified by t
 [role="screenshot"]
 image::images/nav-overview.gif[Overview of the navigation menu]
 
+[float]
+[[inline-actions]]
+== Inline actions for fields and values
+
+Throughout the {security-app}, you can hover over many data fields and values to display inline actions, which allow you to customize your view or investigate further based on that field or value. 
+
+[role="screenshot"]
+image::images/inline-actions-menu.png[Inline additional actions menu,45%]
+
+In some visualizations, these actions are available in the legend by clicking a value's options icon (image:images/three-dot-icon-vertical.png[Vertical three-dot icon,16,16]).
+
+[role="screenshot"]
+image::images/inline-actions-legend.png[Actions in a visualization legend,90%]
+
+Inline actions include the following (some actions are unavailable in some contexts):
+
+* *Filter In*: Add a filter that includes the selected value.
+* *Filter Out*: Add a filter that excludes the selected value.
+* *Add to timeline*: Launch Timeline and populate a query with the selected value.
```

Comment:
We have 2 kinds of actions which add filters to the Timeline:
- **Investigate in Timeline**: Clears the Timeline, adds the cell filter(s) to timeline and automatically opens the timeline. It is used in the Detecion & Response dashboard, and in the Entity Analytics dashboard. (there are also "row actions" in the first column of the alerts/events tables that do the same thing, but those are not CellActions)

- **Add to Timeline**: Adds the cell filter(s) to Timeline without clearing the previous values, and it does not open the timeline. It is used everywhere else, alerts/events table cells, visualizations legends, details flyouts...

@stephmilovic did I miss anything?

Reply:
Thanks for the clarification, @semd! I think we have **Investigate in Timeline** covered correctly in other parts of the docs, and this commit corrects the docs for **Add to timeline**: https://github.com/elastic/security-docs/pull/3296/commits/13bcd00c556d82ea044c538bd6c767dcbace4707